spec/systems/pagination_spec.rb

```diff
@@ -0,0 +1,159 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'pagination', type: :system do
+  ActiveJob::Base.queue_adapter = :test
+
+  context 'given 60 keywords csv' do
+    # 25 keywords per page
+    it 'displays first page' do
+      user = Fabricate(:user)
+      file_path = Rails.root.join('spec', 'fabricators', 'files', '60_keywords.csv')
+
+      visit keywords_path
+
+      within 'form' do
+        fill_in('username', with: user[:username])
+        fill_in('password', with: 'password')
+
+        click_button(I18n.t('auth.login'))
+      end
+
+      attach_file('csv_import_form[file]', file_path)
```

Comment:
similar to [this comment](https://github.com/gutakk/ruby-google-scraper/pull/11#discussion_r507513915), should we create the articles in DB directly instead?

Reply:
Fixed in e265c9410da9c88c674f47280fb981fb94f20dbd